File: MergeNames.cs
using System;
using System.Linq;

public class MergeNames
{
    public static string[] UniqueNames(string[] names1, string[] names2)
    {
        return names1.Union(names2).ToArray();
        
    }
    
    public static void Main(string[] args)
    {
        string[] names1 = new string[] {"Ava", "Emma", "Olivia"};
        string[] names2 = new string[] {"Olivia", "Sophia", "Emma"};
        Console.WriteLine(string.Join(", ", MergeNames.UniqueNames(names1, names2))); // should print Ava, Emma, Olivia, Sophia
    }
}

File: VB.net/MergeNames.vb
Imports System

Public Class MergeNames
    Public Shared Function UniqueNames(ByVal names1 As String(), ByVal names2 As String()) As String()
        Return names1.Union(names2).ToArray()
    End Function

    Public Shared Sub Main(ByVal args As String())
        Dim names1 As String() = New String() {"Ava", "Emma", "Olivia"}
        Dim names2 As String() = New String() {"Olivia", "Sophia", "Emma"}
        Console.WriteLine(String.Join(", ", MergeNames.UniqueNames(names1, names2)))
    End Sub
End Class
